app/models/automated_checks/claim_verifiers/qualifications.rb

```diff
@@ -35,10 +32,23 @@ def complete_match
         create_task(match: :all, passed: true)
       end
 
-      def create_note(field_body = nil)
+      def create_note(match:)
+        body = if dqt_teacher_status.nil?
+          "Not eligible"
+        else
+          <<~HTML
+            #{match == :none ? "Ine" : "E"}ligible:
+            <pre>
+              ITT subject codes: #{dqt_teacher_status.itt_subject_codes}
+              Degree codes:      #{dqt_teacher_status.degree_codes}
+              QTS award date:    #{dqt_teacher_status.qts_award_date}
+            </pre>
+          HTML
```

Comment:
@nomasprime Assume when we do the work you describe in the ticket this will be removed/change?
As said in a previous review this feels presentational and therefore in the wrong place.
Given time constraints and urgency will accept